src/OpenToolkit.Mathematics/Matrix/Matrix3x4.cs

```diff
@@ -985,7 +1011,8 @@ public override bool Equals(object obj)
         /// </summary>
         /// <param name="other">An matrix to compare with this matrix.</param>
         /// <returns>true if the current matrix is equal to the matrix parameter; otherwise, false.</returns>
-        public bool Equals(Matrix3x4 other)
+       [Pure]
+                public bool Equals(Matrix3x4 other)
```

Comment:
wrong indentation

Reply:
Thanks for the look through, this should now be fixed ^^